pkg/cli/interactive_tests/test_demo.tcl

```diff
@@ -10,6 +10,8 @@ eexpect "Welcome"
 eexpect "your changes to data stored in the demo session will not be saved!"
 # Inform the necessary URL.
 eexpect "Web UI: http:"
+# See the notice about insecure mode.
+eexpect "Cockroach demo is running in insecure mode."
```

Comment:
can you add an expect for this message in the two cases of the test below ("check that demo insecure says hello properly") thanks